src/sandbox/mod.rs

```diff
@@ -0,0 +1,68 @@
+#[cfg_attr(target_os = "linux", path = "linux.rs")]
+#[cfg_attr(not(target_os = "linux"), path = "unsupported.rs")]
+mod sandbox_impl;
+
+use pyo3::{create_exception, exceptions::PyException, prelude::*, types::PyTuple};
+
+#[derive(Clone, Debug)]
+pub enum FSAccess {
+    Read(String),
+    ReadWrite(String),
+}
+
+/// Enforces access restrictions
+#[pyfunction(name = "restrict_access", signature=(*rules))]
+fn py_restrict_access(_py: Python, rules: &PyTuple) -> PyResult<()> {
```

Comment:
Out of curiosity, is `_py` underscored because it's unused in the function ?

Reply:
Yes, but it also can be removed.